apps/site/assets/ts/stop/components/DepartureTimes.tsx

```diff
@@ -294,18 +296,40 @@ const DepartureTimes = ({
   const schedules = schedulesByHeadsign(schedulesForDirection);
 
   return (
-    <>
-      {Object.entries(schedules).map(([headsign, schs]) => {
-        const formattedTimes = toDisplayTime(
-          schs,
-          predictionsByHeadsign[headsign]
-            ? predictionsByHeadsign[headsign]
-            : [],
-          overrideDate
-        );
-        return departureTimeRow(headsign, formattedTimes);
-      })}
-    </>
+    <div>
+      {onClick ? (
+        <div
+          className="testSample"
+          onClick={() => onClick(route, directionId, schedulesForDirection)}
+          onKeyDown={() => onClick(route, directionId, schedulesForDirection)}
+          role="presentation"
+        >
+          {Object.entries(schedules).map(([headsign, schs]) => {
+            const formattedTimes = toDisplayTime(
+              schs,
+              predictionsByHeadsign[headsign]
+                ? predictionsByHeadsign[headsign]
+                : [],
+              overrideDate
+            );
+            return departureTimeRow(headsign, formattedTimes);
+          })}
+        </div>
+      ) : (
+        <div>
+          {Object.entries(schedules).map(([headsign, schs]) => {
+            const formattedTimes = toDisplayTime(
+              schs,
+              predictionsByHeadsign[headsign]
+                ? predictionsByHeadsign[headsign]
+                : [],
+              overrideDate
+            );
+            return departureTimeRow(headsign, formattedTimes);
+          })}
```

Comment:
suggestion (non-blocking): Consider splitting this out into a helper element on the page to avoid duplication